controllers/usersignup/usersignup_controller_test.go

```diff
@@ -2764,103 +2951,129 @@ func TestUserSignupListBannedUsersFails(t *testing.T) {
 		})
 }
 
-func TestUserSignupDeactivatedButMURDeleteFails(t *testing.T) {
-	// given
-	userSignup := &toolchainv1alpha1.UserSignup{
-		ObjectMeta: NewUserSignupObjectMeta("", "alice.mayweather.doe@redhat.com"),
-		Spec: toolchainv1alpha1.UserSignupSpec{
-			Userid:   "UserID123",
-			Username: "alice.mayweather.doe@redhat.com",
-			States:   []toolchainv1alpha1.UserSignupState{toolchainv1alpha1.UserSignupStateDeactivated},
+func TestUserSignupDeactivatedButMUROrSpaceDeleteFails(t *testing.T) {
+	for testname, testValues := range map[string]map[string]string{
+		"usersignup deactivated but mur delete failed": {
+			"Reason":  "UnableToDeleteMUR",
+			"Message": "unable to delete mur",
 		},
-		Status: toolchainv1alpha1.UserSignupStatus{
-			Conditions: []toolchainv1alpha1.Condition{
-				{
-					Type:   toolchainv1alpha1.UserSignupComplete,
-					Status: v1.ConditionTrue,
+		"usersignup deactivated but space delete failed": {
+			"Reason":  "UnableToDeleteSpace",
+			"Message": "unable to delete space",
+		},
+	} {
+		t.Run(testname, func(t *testing.T) {
```

Comment:
I would prefer this format - it's easier to read/understand:
```go
	type testCases struct {
		name, reason, message string
	}
	cases := []testCases{
		{
			name:    "usersignup deactivated but mur delete failed",
			reason:  "UnableToDeleteMUR",
			message: "unable to delete mur",
		},
		{
			name:    "usersignup deactivated but space delete failed",
			reason:  "UnableToDeleteSpace",
			message: "unable to delete space",
		}}
	for _, testCase := range cases {
		t.Run(testCase.name, func(t *testing.T) {
...
```

Reply:
Done as part of 0c6cd71f6716ef1978c21a9a087fc83cf4f35632